scripts/populate_tox/populate_tox.py

```diff
@@ -135,6 +140,10 @@ def _prefilter_releases(integration: str, releases: dict[str, dict]) -> list[Ver
         if meta["yanked"]:
             continue
 
+        if older_than is not None:
+            if datetime.fromisoformat(meta["upload_time_iso_8601"]) > older_than:
+                continue
```

Comment:
[nit – feel free to ignore]

```suggestion
        if older_than is not None and datetime.fromisoformat(meta["upload_time_iso_8601"]) > older_than:
            continue
```